packages/console/src/onboarding/pages/SignInExperience/sie-config-templates.ts

```diff
@@ -22,7 +22,7 @@ const configTemplate1: OnboardingSieConfig = {
 const configTemplate2: OnboardingSieConfig = {
   logo: `${assetsUrl}/kvCN1Z2a/logo2.png`,
   color: '#F47346',
-  identifier: SignInIdentifier.Phone,
+  identifier: SignInIdentifier.Username,
   authentications: [Authentication.Password, Authentication.VerificationCode],
```

Comment:
Remove verification code from authentications as well?

Reply:
remove all phone identifier and revisit later.